playbooks/tasks/portal-role-task-mongo-setup-docker-service.yml

```diff
@@ -84,7 +83,8 @@
         group: "{{ webportal_user }}"
         remove: True
       become: True
-    
+      when: mgkey_stat_result.stat.exists
```

Comment:
I added this as a stop gap, but there is probably a better fix. We shouldn't be triggering this block at all on new servers when nothing exists. A user shouldn't have to tell ansible to delete the mongo database on a new server set up.